packages/localizations/src/sv-SE.ts

```diff
@@ -99,31 +100,31 @@ export const svSE: LocalizationResource = {
   footerPageLink__privacy: 'Integritet',
   footerPageLink__terms: 'Villkor',
   formButtonPrimary: 'Fortsätt',
-  formButtonPrimary__verify: 'Verify',
+  formButtonPrimary__verify: 'Verifiera',
   formFieldAction__forgotPassword: 'Glömt lösenord?',
   formFieldError__matchingPasswords: 'Lösenorden matchar.',
   formFieldError__notMatchingPasswords: 'Lösenorden matchar inte.',
   formFieldError__verificationLinkExpired: 'Verifieringslänken har löpt ut. Vänligen begär en ny länk.',
   formFieldHintText__optional: 'Valfritt',
   formFieldHintText__slug: 'En slug är ett läsbart ID som måste vara unikt. Det används ofta i URL:er.',
-  formFieldInputPlaceholder__backupCode: undefined,
+  formFieldInputPlaceholder__backupCode: 'Ange din reservkod',
   formFieldInputPlaceholder__confirmDeletionUserAccount: 'Radera konto',
-  formFieldInputPlaceholder__emailAddress: undefined,
-  formFieldInputPlaceholder__emailAddress_username: undefined,
+  formFieldInputPlaceholder__emailAddress: 'Ange din e-postadress',
+  formFieldInputPlaceholder__emailAddress_username: 'Ange din e-postadress eller ditt användarnamn',
   formFieldInputPlaceholder__emailAddresses:
     'Ange eller klistra in en eller flera e-postadresser, separerade med mellanslag eller kommatecken',
-  formFieldInputPlaceholder__firstName: undefined,
-  formFieldInputPlaceholder__lastName: undefined,
-  formFieldInputPlaceholder__organizationDomain: undefined,
-  formFieldInputPlaceholder__organizationDomainEmailAddress: undefined,
-  formFieldInputPlaceholder__organizationName: undefined,
-  formFieldInputPlaceholder__organizationSlug: undefined,
-  formFieldInputPlaceholder__password: undefined,
-  formFieldInputPlaceholder__phoneNumber: undefined,
-  formFieldInputPlaceholder__username: undefined,
-  formFieldLabel__automaticInvitations: 'Enable automatic invitations for this domain',
-  formFieldLabel__backupCode: 'Reserv-kod',
-  formFieldLabel__confirmDeletion: 'Confirmation',
+  formFieldInputPlaceholder__firstName: 'Ange ditt förnamn',
+  formFieldInputPlaceholder__lastName: 'Ange ditt efternamn',
+  formFieldInputPlaceholder__organizationDomain: 'Ange organisationsdomän',
+  formFieldInputPlaceholder__organizationDomainEmailAddress: 'Ange e-postadress för verifiering',
+  formFieldInputPlaceholder__organizationName: 'Ange organisationsnamn',
+  formFieldInputPlaceholder__organizationSlug: 'min-organisation',
+  formFieldInputPlaceholder__password: 'Ange ditt lösenord',
+  formFieldInputPlaceholder__phoneNumber: 'Ange ditt telefonnummer',
+  formFieldInputPlaceholder__username: 'Ange ditt användarnamn',
+  formFieldLabel__automaticInvitations: 'Aktivera automatiska inbjudningar för denna domän',
+  formFieldLabel__backupCode: undefined,
```

Comment:
Let's revert this, if it makes sense.

Reply:
I reverted 'formFieldLabel__backupCode' and kept the rest.